web/api/report_server.thrift

```diff
@@ -325,6 +325,7 @@ struct ReportData {
   // of custom labels that describe some properties of a report. For example the
   // timestamp in case of dynamic analyzers when the report was actually emitted.
   18: optional map<string, string> annotations,
+  19: optional BlameInfo blameInfo,    // Blmae info.
```

Comment:
Its a typo, but more importantly, maybe this isn't the comment we need here :) How about
"Contains the git blame information of the report. May be NULL if the analysis was _not_ done in a git repository."

Reply:
Thanks for your suggestion! I completely agree. It would be better if there was a more precise description.